python/lsst/daf/butler/script/queryDatasets.py

```diff
@@ -195,7 +195,7 @@ def getTables(self) -> list[AstropyTable]:
                 tables[dataset_ref.datasetType.name].add(dataset_ref)
         else:
             d = list(self.datasets)
-            ref_uris = self.butler.datastore.getManyURIs(d, predict=True)
+            ref_uris = self.butler._datastore.getManyURIs(d, predict=True)
```

Comment:
I imagine we need to create a `butler.get_many_URIs()` method and switch to that here.